HerPublicWebsite/Views/Questionnaire/SelectLocalAuthority.cshtml

```diff
@@ -20,54 +20,77 @@
         
         <h1 class="govuk-heading-l">Select your local authority</h1>
         
-        <div class="her_alphabet_links">
-            @{
-                var orderedLetters = Model.LocalAuthoritiesByInitial.Keys.OrderBy(k => k).ToList();
-                var lettersInFirstRow = (orderedLetters.Count + 1) / 2;
```

Comment:
Losing this makes it work a bit better when there are only a few letters, but it messes up the layout when there isn't a filter value.